### PR TITLE
[12.0][IMP] partner_risk_insurance: Add tracking to company_credit_limit and insurance_credit_limit on partners

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT28073

please @sergio-teruel @carlosdauden  review this

### CODE REVIEW

**Review by marcelsavegnago — APPROVED** [review]

File: partner_risk_insurance/models/res_partner.py [hunk 1]
```diff
@@ -14,12 +14,16 @@ def _compute_credit_limit(self):
...
-    company_credit_limit = fields.Float("Company's Credit Limit",
-                                        help='Credit limit granted by the '
-                                        'company.')
-    insurance_credit_limit = fields.Float("Insurance's Credit Limit",
-                                          help='Credit limit granted by the '
-                                          'insurance company.')
+    company_credit_limit = fields.Float(
+        "Company's Credit Limit",
+        help='Credit limit granted by the company.',
+        tracking=True,
```
Comment (Tardo) [inline partner_risk_insurance/models/res_partner.py]: You mean 'track_visibility'?

File: partner_risk_insurance/models/res_partner.py [hunk 2]
```diff
@@ -14,12 +14,16 @@ def _compute_credit_limit(self):
...
+    )
+    insurance_credit_limit = fields.Float(
+        "Insurance's Credit Limit",
+        help='Credit limit granted by the insurance company.',
+        tracking=True,
```
Comment (Tardo) [inline partner_risk_insurance/models/res_partner.py]: You mean 'track_visibility'?

**Review by Tardo — CHANGES_REQUESTED** [review]

**Comment by CarlosRoca13** [comment]
@Tardo yes, was a confusion

**Review by Tardo — APPROVED** [review]

**Pull request merged**